pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadatainterfaces/helpers.go

```diff
@@ -14,17 +14,17 @@ const UnknownOwner = "Unknown"
 
 var (
```

Comment:
I am afraid that I am missing the point.
Isn't /var/lib/kubelet/pki/kubelet-client-current.pem covered as a on-disk cert?

Reply:
Ah, good catch - we don't scan this directory. So far its limited to [this](https://github.com/openshift/origin/blob/d347e6a472dc3756ccb2a21f83dd0a2a65028c98/test/extended/operators/manifests/pod.yaml#L13-L18) list